src/backend/main.ts

```diff
@@ -580,7 +580,7 @@ ipcMain.on('unlock', () => {
     unlinkSync(join(gamesConfigPath, 'lock'))
     if (powerId) {
       logInfo('Stopping Power Saver Blocker', LogPrefix.Backend)
-      return powerSaveBlocker.stop(powerId)
+      powerSaveBlocker.stop(powerId)
```

Comment:
any specific reason for the return removal?

Reply:
it gives a yarn codecheck error now on the latest version of electron (i.e. "not all codepaths return a value"). it is an ipcMain.on handler, so it doesn't return anything anyways